devops.rst

```diff
@@ -13,4 +13,5 @@ If you plan to use Conan in production in your project, team, or organization, t
    :maxdepth: 1
 
    devops/using_conancenter
+   devops/sources_backup
```

Comment:
I am confused about ``devops/versioning`` section here, this doesn't exist yet in ``release/2.0`` branch?

Reply:
`devops/versioning` is already in release2.0 yes, it contains the prerelease docs for now